public/js/components/includes/Navigation.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { IndexLink, Link } from "react-router";
+export default class Navigation extends React.Component{
+    constructor(){
```

Comment:
constructor usually use 1 parameter as props, so please add this parameter to constructor and super()